game/src/client/scenes/raceGame/RaceGameEvents.ts

```diff
@@ -16,6 +16,9 @@ export const EventNames = {
 	followPlayerToggle: "follow-player-toggle",
 	throwingBananaToggle: "throwing-banana-toggle",
 
+	zoomIn: "zoom-in",
+	zoomOut: "zoom-out",
```

Comment:
Je vois qu'avec des boutons + et - pour le zoom, ces events ont bien leur place. Par contre, est-ce que d'avoir des events comme ça pour le Zoom in et le Zoom out est compatible avec un Zoom controllé par la Mouse Wheel ou un Pinch (multitouch sur écran tactile)?